spec/requests/api/v1/moves_controller_create_spec.rb

```diff
@@ -87,6 +89,10 @@
         expect(move.documents).to eq([document])
       end
 
+      it 'associates a reason with the newly created move' do
+        expect(move.reason).to eq(reason)
+      end
+
```

Comment:
sure we do need this test?
should this be implicit in other tests?

Reply:
Yes! I was worried that we didn't associate a reason with a move when I first read the code :-)